test/topology_custom/test_tablets.py

```diff
@@ -23,3 +23,55 @@ async def test_tablet_change_replication_vnode_to_tablets(manager: ManagerClient
     with pytest.raises(InvalidRequest):
         await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1, 'initial_tablets': 1};")
 
+
+@pytest.mark.asyncio
+async def test_tablet_change_replication_strategy(manager: ManagerClient):
+    cfg = {'enable_user_defined_functions': False,
+           'experimental_features': ['tablets', 'consistent-topology-changes']}
+    server = await manager.server_add(config=cfg)
+
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};")
+    await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy'};")
+
+    res = await cql.run_async("DESCRIBE KEYSPACE test")
+    assert "NetworkTopologyStrategy" in res[0].create_statement, "NetworkTopologyStrategy wasn't switched onto"
+
+    res = await cql.run_async("SELECT * FROM system_schema.scylla_keyspaces WHERE keyspace_name = 'test'")
+    assert len(res) == 0, "tablets replication strategy turned on"
+
+
+@pytest.mark.asyncio
+async def test_tablet_default_initialization(manager: ManagerClient):
+    cfg = {'enable_user_defined_functions': False,
+           'experimental_features': ['tablets', 'consistent-topology-changes']}
+    server = await manager.server_add(config=cfg)
+
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1, 'initial_tablets': 1};")
```

Comment:
You didn't explain in a comment what this test is about (I really recommend to always do that...) but its name suggests it is about **default** initialization, and yet the CREATE KEYSPACE command is explicitly setting initial_tablets = 1 and is NOT about the default. What am I missing?

Reply:
Right now the "initial_tablets: <number>" is the only way to turn tablets on. So what the "default" means here is -- what should be visible once tablets are turned on. After #16364 it will be more self-contained, I agree

Reply:
I see. I would have written this in a FIXME comment because it's impossible to guess. But since I guess you plan to change this test anyway, never mind now. Please note that if "initial_tablets" is still supported, you'll probably never remember to go back to fix this test, and it will never be about "the default" unless you remember to go back to this test and remove the explicit "initial_tablets" setting.